src/smtencoding/FStar.SMTEncoding.Term.fs

```diff
@@ -408,6 +413,7 @@ let mkSub = mk_bin_op Sub
 let mkDiv = mk_bin_op Div
 let mkMul = mk_bin_op Mul
 let mkMod = mk_bin_op Mod
+let mkRealOfInt t r = mkApp ("to_real", [t]) r
```

Comment:
This seems to be the only new addition in terms of Z3 primitive functions, otherwise the binops etc. are same as those for integers, right?

Reply:
The other functions are overloaded by sort in Z3